misc/custom.scss

```diff
@@ -1,48 +1,33 @@
 ---
+
 ---
+
 @charset "utf-8";
 
-// HLB brand colors
-$primary: #363a65;
-$secondary: #6b4370;
-$secondary-hover: #1fbbac;
-$table-grey: #F0F4F6;
+// Agency brand colors
+$primary: {{- site.colors.primary-color | default: '#f5a623' -}};
+$secondary: {{- site.colors.secondary-color | default: '#50e3c2' -}};
```

Comment:
are we not removing the default here?

Reply:
Removed in https://github.com/isomerpages/isomerpages-template/pull/312/commits/0694a51f93680dee108fe465917f66e68c539f90!